RNSound/RNSound.m

```diff
@@ -79,7 +79,7 @@ -(void) audioPlayerDidFinishPlaying:(AVAudioPlayer*)player
     [self setOnPlay:NO forPlayerKey:key];
     RCTResponseSenderBlock callback = [self callbackForKey:key];
     if (callback) {
-      callback(@[@(flag)]);
+      callback([NSArray arrayWithObjects:[NSNumber numberWithBool:flag], nil]);
```

Comment:
Not a blocker, but there's a recommendation in the Clang 7 docs to use the [NSArray arrayWithObjects:count:] form instead of the variadic nil-terminated form. Excerpt:
```
Using array and dictionary literals is safer than the variadic creation forms commonly in use today. Array literal expressions expand to calls to +[NSArray arrayWithObjects:count:], which validates that all objects are non-nil. The variadic form, +[NSArray arrayWithObjects:] uses nil as an argument list terminator, which can lead to malformed array objects. 
```